From 37eba4fbd008a92581ece712904b93a97c018224 Mon Sep 17 00:00:00 2001
From: "Huang, Tao" <huangtao@rock-chips.com>
Date: Fri, 25 Mar 2016 11:37:14 +0800
Subject: [PATCH] Revert "ARM64: dts: rk3399: add pmu node"

This reverts commit 0b622df3499e3dcaebd619fa966b13588131b83d.

arm pmu driver do not support PPI in two
cluster well. So drop it.

Change-Id: I69f43ad1703589805c7e86749badda8bf802d51a
---
 arch/arm64/boot/dts/rockchip/rk3399.dtsi | 16 ----------------
 1 file changed, 16 deletions(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3399.dtsi b/arch/arm64/boot/dts/rockchip/rk3399.dtsi
index 85c03abdcb22..c03d4774554a 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3399.dtsi
@@ -221,22 +221,6 @@
 			     <GIC_PPI 10 IRQ_TYPE_LEVEL_LOW>;
 	};
 
-	pmu_a53 {
-		compatible = "arm,cortex-a53-pmu";
-		interrupts = <GIC_PPI 7 IRQ_TYPE_LEVEL_LOW>;
-		interrupt-affinity = <&cpu_l0>,
-				     <&cpu_l1>,
-				     <&cpu_l2>,
-				     <&cpu_l3>;
-	};
-
-	pmu_a72 {
-		compatible = "arm,cortex-a72-pmu", "arm,cortex-a57-pmu";
-		interrupts = <GIC_PPI 7 IRQ_TYPE_LEVEL_LOW>;
-		interrupt-affinity = <&cpu_b0>,
-				     <&cpu_b1>;
-	};
-
 	xin24m: xin24m {
 		compatible = "fixed-clock";
 		#clock-cells = <0>;
-- 
2.35.3

